docs/embedding/start.md

```diff
@@ -18,9 +18,9 @@ You'll embed the full Metabase application in your app. Once logged in, people c
 
 ## [Static embedding](./static-embedding.md)
 
-Also known as Signed Embedding, Static embedding offers drill-through with custom destinations, so you can define what happens when people click on a chart, like sending people to another chart or URL–all while securing the underlying data.
+Also known as Signed Embedding, Static embedding is a secure way to embed individual charts and dashboards. Your users will see data, but won’t be able to click through into it.
```

Comment:
```suggestion
Also known as Signed Embedding, Static embedding is a secure way to embed charts and dashboards. Static embeds are view only; people won't be able to drill-through charts and tables.
```

Edit prefers "people" to "users", and emphasizes static embed's view-only nature.